### PR TITLE
[14.0][ADD] module currency_old_rate_notify

### PR DESCRIPTION
**Pull request by alexis-via**
This module uses web_notify to send a notification to accounting managers when currency rates are older than 3 days (3 days is configurable via an ir.config_parameter).

See https://github.com/OCA/currency/issues/136 for the discussion about the design of the module.

notify_old_rates

### CODE REVIEW

**Comment by desdelinux** [comment]
@alexis-via I think you should add a parameter to exclude currencies

File: currency_old_rate_notify/models/res_currency_rate.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        return max_days
+
+    @api.model
+    def notify_rates_too_old(self):
+        max_days = self._notify_rates_get_max_days()
+        today = fields.Date.context_today(self)
+        limit_date = today - timedelta(max_days)
+        logger.debug("notify_rates_too_old limit_date=%s", limit_date)
+        companies = self.env["res.company"].search([])
+        currencies = self.env["res.currency"].search([])
```
Comment (paulius-sladkevicius) [inline currency_old_rate_notify/models/res_currency_rate.py]: As it was stated, I agree that needs to filter only active currencies.

**Review by paulius-sladkevicius — CHANGES_REQUESTED** [review]

**Comment by paulius-sladkevicius** [comment]
Great addon - looking forward to trying to use it in production.

**Review by nibraun — APPROVED** [review]
Successfully used in prod

**Review by desdelinux — APPROVED** [review]

File: currency_old_rate_notify/models/res_currency_rate.py (hunk 1 above)
Comment (alexis-via) [reply currency_old_rate_notify/models/res_currency_rate.py]: that's exactly what the code does :)

**Review by paulius-sladkevicius — APPROVED** [review]

**Comment by andrius-preimantas** [comment]
Great module. Any chances having it merged to master? Would like to migrate to v13

**Comment by klodr** [comment]
Tested and approved 
please merge

**Pull request merged**